subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/junit/JUnitJdkNavigationTest.groovy

```diff
@@ -0,0 +1,26 @@
+package org.gradle.testing.junit
+
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+import org.gradle.integtests.fixtures.DefaultTestExecutionResult
+
+class JUnitJdkNavigationTest extends AbstractIntegrationSpec {
```

Comment:
We will need to verify that the change also works for TestNG. Would you mind adding a test class for it as well?